llvm/test/CodeGen/AMDGPU/use-after-free-after-cleanup-failed-vreg.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: not llc -mcpu=gfx1100 -mtriple=amdgcn-amd-amdhsa -stress-regalloc=4 %s 2>&1 | FileCheck %s
+
+; CHECK: ran out of registers during register allocation in function 'f'
```

Comment:
Include the error prefix, and can you add an implicit check not to cover other errors? Want to be sure the machine verifier doesn't fire

Reply:
Oh I figured. No `Bad machine code`.